File: client/src/app/viewmodels/OpenChallenge.ts
import { Injectable } from '@angular/core';

import { environment } from '../../environments/environment';


export class OpenChallenge {
    Category: string;
    Currency: string;
    Reward: number;
    Buyin: number;
    Description: string;
    Author: string;
    AuthorId: string;
    CreatedAt: string;
}

File: server/Model/Domain/OpenChallenge.cs
using System;
using System.Collections.Generic;
using Microsoft.AspNetCore.Http;

namespace webapi.Model.Domain
{
    public class OpenChallenge
    {
        public string Category { get; set; }
        public string Currency{ get;set; }
        public decimal Reward{ get;set; }
        public decimal Buyin { get;set; }
        public string Description { get; set; }
        public string Author { get; set; }
        public string AuthorId{ get;set; }
        public DateTimeOffset CreatedAt { get; set; }
    }
}